### PR TITLE
[WIP] Implement deploy-to-staging step

### PR DESCRIPTION
**Pull request by izumin5210**
- [x] Deploy docker image to arukas.io
- [x] Notify to slack / github
- [x] Basic auth
- [x] Disallow crawlers
  - should put `robots.txt` to the docker image
- [ ] Remove arukas app when it was merged

### CODE REVIEW

**Comment by izumin5210** [comment]
test comment

**Comment by izumin5210** [comment]
test comment

**Comment by izumin5210** [comment]
[BOT] Finish deployment to the staging :rocket:
https://wakate-org-70.arukascloud.io

**Comment by izumin5210** [comment]
[BOT] Finish updating the staging :up:
https://wakate-org-70.arukascloud.io

**Comment by izumin5210** [comment]
[BOT] Finish updating the staging :up:
https://wakate-org-70.arukascloud.io

**Comment by izumin5210** [comment]
[BOT] Finish deployment to the staging :rocket:
https://wakate-org-70.arukascloud.io

File: scripts/deploy-to-staging.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        --instances=1 \
+        --mem=256 \
+        --ports=80:tcp \
+        --app-name="$appName" \
+        --name="$appName" \
+        "$imageName"
+
+    appId=$(./arukas ps --all | grep $appName | awk '{ print $1 }')
+    printHeader "Start arukas app (id: $appId, name: $appName)"
+    ./arukas start $appId
```
Comment (izumin5210) [inline scripts/deploy-to-staging.sh]: `arukas run` seems to start the created container automatically... 💭  https://github.com/arukasio/cli/blob/v0.1.3/run.go#L54

**Comment by kyontan** [comment]
Arukasはサービス終了しました 😭

**Pull request closed without merging**